test/integration/upgrade-command/first-operator/operator.yaml

```diff
@@ -11,7 +11,7 @@ tasks:
     kind: Apply
     spec:
       resources:
-        - pod.yaml
+        - configmap.yaml
```

Comment:
We used to ignore `Pod`s health status (they were healthy by default). This is not the case anymore so we need to use another kind of object here that is "healthy".